lib/menu.rb

```diff
@@ -0,0 +1,33 @@
+class Menu
+  def initialize(items = [])
+    @items = items
+  end
+
+  def items
+    return @items
+  end
+
+  def add_item(name, price)
+    @items << new_food(name, price)
```

Comment:
good use of SRP, by separating the actual creation of the new dish down below in a private method